fatips/300.md

```diff
@@ -0,0 +1,438 @@
+         "salt":"i2n3f8923n98f2981sd701eed29360234234n2jnb5",
+          "extIds":[
+           "fb736f345dd5d67ac26eab94c9c9f51b708b05c",
+           "qifu3ino28381n2937n1293mi912kfmlasfkasdkmfasdfasdf",
+           "f2on3p2u123n712n387nf19msov2jbmurbhimsdo",
+           "mvnansuvbeybqyyyewvyqv273212oi3onbvttbywtybhxzzxuv",
+       ]
+      }
+   ]
+}
+```
+
+(note: extIds shown are not realistic values)
+
+<br/>
+
+### `get-balance` :
+
+Get the balance of an address for a token
+
+#### Parameters:
+
+| Name         | Type   | Description                               | Validation                      | Required |
+| ------------ | ------ | ----------------------------------------- | ------------------------------- | -------- |
+| `fa-address` | string | The Factoid address to get the balance of | Must be a valid Factoid address | Y        |
+
+#### Response:
+
+```json
+{  
+   "jsonrpc":"2.0",
+   "id":1,
+   "result": 2321111
+}
+```
+
+<br/>
+
+### `get-stats` :
+
+Get overall statistics for a token
+
+#### Parameters:
+
+| Name | Type | Description | Validation | Required |
+| ---- | ---- | ----------- | ---------- | -------- |
+|      |      |             |            |          |
+
+#### Response:
+
+```json
+{  
+   "jsonrpc":"2.0",
+   "id":1,
+    "result":{
+        "supply":10000000,
+        "circulatingSupply":8113,
+        "transactions":18429,
+        "issuanceTimestamp": 1518286500,
+        "lastTransactionTimestamp": 1539212767
+    }
+}
+```
+
+
+
+### `get-nf-token` :
+
+Get a non fungible token by ID. The token belong to non fungible token class.
+
+#### Parameters:
+
+| Name          | Type   | Description                             | Validation                                 | Required |
+| ------------- | ------ | --------------------------------------- | ------------------------------------------ | -------- |
+| `nf-token-id` | string | The unique ID of the non fungible token | Must be a valid issued non fungible token. | Y        |
+
+#### Response:
+
+```json
+{  
+   "jsonrpc":"2.0",
+   "id":1,
+    "result":{
+    	"id": "0",
+        "metadata":{
+            "name": "Mr. Cucumber"
+        }
+    }
+}
+```
+
+
+
+### `send-transaction`:
+
+Send A FAT transaction to a token
+
+#### Parameters:
+
+| Name     | Type   | Description                                                  | Validation                                                   | Required |
+| -------- | ------ | ------------------------------------------------------------ | ------------------------------------------------------------ | -------- |
+| `tx`     | object | The fully formed & signed FAT transaction JSON               | Must conform to all transaction validation criteria of the token's spec | Y        |
+| `extids` | array  | The array of hex encoded ed25519 signatures & RCDs (extids) for each transaction input | Must be valid Signatures & RCDS as per the transaction implementation of the token spec. | Y        |
```

Comment:
This parameter shouldn't be named extids. This is a common pitfall to name things based on how they are going to be used instead of what they are doing (I wish I could insert a relevant article here but I cannot find one rn). You didn't name the first parameter `content`. At the API level you should be aware of how this data is going to be used. So it should be name something like "signaturesRcds" or along those lines

Reply:
Thanks @PaulBernier, good point